packages/npm-vue/src/index.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Atom,
+  AtomMut,
+  Ctx,
+  CtxSpy,
+  createCtx,
+  throwReatomError,
+} from '@reatom/core'
+import { App, ref, Ref, onScopeDispose, inject, InjectionKey } from 'vue'
+
+const reatomCtxKey = Symbol('reatomCtxKey') as InjectionKey<Ctx>
```

Comment:
If it is not required to use a Symbol, let's use a string key. Additionally it is good to add a check that the provider initialized only once